src/services/homework/hooks/submissions.js

```diff
@@ -303,6 +303,7 @@ const hasDeletePermission = (hook) => {
 exports.before = () => ({
 	all: [authenticate('jwt'), stringifyUserId],
 	find: [
+		// todo restrictToCurrentSchool,
```

Comment:
o.O Can we not just append the user's schoolId to every query?

Reply:
At the moment only with additional request. But it is a follow-up.
https://ticketsystem.schul-cloud.org/browse/SC-3740